qunicorn_core/db/models/provider.py

```diff
@@ -27,6 +27,7 @@ class ProviderDataclass(DbModel):
     """Dataclass for storing Providers
 
     Attributes:
+        id (int): The id of a provider
```

Comment:
'.' at the end of the sentence
OR:
Discussion: should we end each description with a '.' ? -> if so it should be done consistently

Reply:
I put now a point at the end of each one